src/index.ts

```diff
@@ -278,6 +300,9 @@ export async function install(
   };
   const packageBundle = await rollup.rollup(inputOptions);
   await packageBundle.write(outputOptions);
+  assets.forEach(asset => {
+    fs.copyFileSync(asset, `${destLoc}/${path.basename(asset)}`);
```

Comment:
The way that "webDependencies" paths currently behave is to install `node_modules/FULL_PATH` -> `web_modules/FULL_PATH`. Currently this PR is installing assets to `web_modules/FILE` instead. This works for simple examples, but can cause conflicts in larger projects (esp. if the filename is abstract like `index.css`).  

I'd recommend using a similar flow to `depObject`:
```
assetObject[depName] = assetLoc;
// ...
Object.entries(assetObject).forEach(([assetName, assetLoc]) => {
  fs.copyFileSync(assetLoc, `${destLoc}/${assetName}`);
}
```

Or, skipping the Object.entries() bit and just saving them in an array to start with:

```
assets.push([depName, assetLoc]);
// ...
assets.forEach(([assetName, assetLoc]) => {
  fs.copyFileSync(assetLoc, `${destLoc}/${assetName}`);
}
```

Reply:
This might be a bit brute-forcey, but what about something like 
```javascript
assets.forEach(asset => {
  const assetPath = path.relative(`${cwd}/node_modules`, asset);
  const assetDirs = path.dirname(assetPath).split('/');
  const built = [];
  assetDirs.forEach(dir => {
    built.push(dir);
    const targetDir = `${destLoc}/${built.join('/')}`;
    if (!fs.existsSync(targetDir)) {
      fs.mkdirSync(targetDir);
    }
  });
    
  fs.copyFileSync(asset, `${destLoc}/${assetPath}`);
});
```

Reply:
I totally get what you mean, but from a maintenance perspective I'd rather that we match the existing model vs. introducing and managing two different ways of tracking files. Does that make sense?

Reply:
I understand the sentiment; however, as far as I know Rollup cannot handle non-JS entry points (there might be a plugin that can set this up), but if you pass in a CSS file or a font, Rollup will error out. 

Maybe I'm missing something here. There are npm packages that are built to do this sort of file copying that could be utilized which would push the copy logic into another utility (or the function above could be abstracted into a separate file, but I'm not sure we can reliably get Rollup to do the moving for non-JS files.

Reply:
Oh yea, I think I'm not being clear. Let me show you what I mean, 1 sec (I believe Github will let me push a change to your branch).

Reply:
Okay, change pushed. Does that make sense? LMK if I'm missing something obvious as well.

Reply:
I haven’t had a chance to run this, but I think this will throw because the directory doesn’t exist, hence the loop in my comment earlier.

Reply:
Ah right, of course. In that case I'd just suggest using https://www.npmjs.com/package/mkdirp, it's used in other Pika tools and is pretty straightforward. 

Once Node v8 is out of LTS, we can update it to https://nodejs.org/api/fs.html#fs_fs_mkdir_path_options_callback with `{recursive: true}` without checking if it exists beforehand. 

In either case, if the directory already exists, it's a noop.

Reply:
Sounds good. I'm well aware of `mkdirp`, but was trying to avoid the dependency. I'll update this tomorrow morning. Sorry for all the back-and-forth and thanks for the help!

Reply:
@FredKSchott I think this latest commit should do it.

Reply:
I think that's just about it! I'm going to merge it and make one small change to add `mkdirp`, since fs.mkdir will throw an error if the directory already exists, and I don't believe will work recursively.

Reply:
Perfect. This does have `mkdirp`:

```javascript
import { sync as mkdirSync } from 'mkdirp';
```

Reply:
🤦‍♂ Doh, for some reason i skimmed that as `from 'fs'`, which as I say out loud still wouldn't have made any sense. Thanks for the correction